app/models/supported_ssg.rb

```diff
@@ -17,31 +17,32 @@ def ref_id
 
   class << self
     def supported?(ssg_version:, os_major_version:, os_minor_version:)
-      ssg_version == ssg_for_os(os_major_version, os_minor_version)
+      versions = ssg_versions_for_os(os_major_version, os_minor_version)
+      versions&.include?(ssg_version)
     end
 
-    def ssg_for_os(os_major_version, os_minor_version)
+    def ssg_versions_for_os(os_major_version, os_minor_version)
       raw_supported.dig(
         'supported',
-        "#{self::OS_NAME}-#{os_major_version}.#{os_minor_version}",
-        'version'
-      )
+        "#{self::OS_NAME}-#{os_major_version}.#{os_minor_version}"
+      )&.map { |raw_ssg| raw_ssg['version'] }
```

Comment:
Nit: We could remove the need for safe operators here and anywhere else we use this in the future.

```suggestion
      )&.map { |raw_ssg| raw_ssg['version'] } || []
```

Reply:
I thought the same. I wanted to convert it to use non-raw attributes, but then got hold off because of the conversion from int to str.

Reply:
Changed to use parsed objects. It'd allow to add more search if we'd add a `supported` flag.